packages/components/src/components/video/video.test.ts

```diff
@@ -0,0 +1,137 @@
+import { expect, fixture, html, waitUntil } from '@open-wc/testing';
+import sinon from 'sinon';
+import type SdVideo from './video';
+
+const defaultSlot = html`<video controls>
+  <source src="http://media.w3.org/2010/05/sintel/trailer.mp4" type="video/mp4" />
+  Your browser does not support the video tag.
+</video>`;
+const posterSlot = html`<img
+  slot="poster"
+  alt="poster"
+  src="https://www.blender.org/wp-content/uploads/2020/10/robin-tran-redautumnforest_pr1.jpg"
```

Comment:
Especially here: Definitely remove external dependencies. If needed just inline the image with base64 or whatever.

Reply:
"just inline the image with base64 or whatever" --- please expand on what you mean.  I don't think you want:
![image](https://github.com/solid-design-system/solid/assets/39494579/507211f3-205b-4e6d-abba-e34989ae0cfc)
I have removed the src completely, it still passes the tests.  The video remains in the tests and is an external dependency.  If you have a better way to do it, please explain, otherwise I'll leave it like this